pkg/backend/display/testdata/not-truncated/failed-stacks-dont-rewrite-operations.json.interactive-100x80-plain.txt

```diff
@@ -1,10 +1,10 @@
 View in Browser (Ctrl+O): link
 
-<%hide-cursor%>
     Type                               Name      Status                  Info<%clear-to-end%>
-     pulumi:pulumi:Stack                test-dev                          1 mes<%clear-to-end%>
- --  ├─ pulumi-python:dynamic:Resource  test-r    deleted original        <%clear-to-end%>
- ~   └─ pulumi-python:dynamic:Resource  test-x    **updating failed**     <%clear-to-end%>
-<%clear-to-end%>
+     Type                               Name      Status                  Info
```

Comment:
Not shown: there is an `\r` in the raw diff that was removed. This character is visible via `git diff` or in an editor that shows control characters.